### DISCUSSION TITLE
我不知道为什么我的forge端进不了服务器

### DISCUSSION THREAD

**Discussion by VeryFuns** [Meloong-Git/PCL#1878] [提问]
我不知道为什么我的forge端进不了服务器 
显示 disc....

**Comment by WForst-Breeze** [comment]
请上传相关截图或错误信息😕谢谢

**Reply by VeryFuns** [reply]
[Uploading QQ录屏20230521135927.zip…]()

**Reply by VeryFuns** [reply]
谢谢QWQ

**Reply by WForst-Breeze** [reply]
想再次确认一下：您是在使用正版账户时尝试使用 Forge 端登录服务器出现此问题，但使用原版进入就无此问题吗🤔

**Reply by VeryFuns** [reply]
是的 除了forge端都可以正常进入服务器

**Comment by so1ve** [comment]
请上截图
看你的报错信息应该是disconnected from server，检查一下你的网络有没有问题

**Reply by VeryFuns** [reply]
![20230521135957](https://github.com/Hex-Dragon/PCL2/assets/134185011/e1664d94-ca3a-463a-a10d-bd83e2994a44)
x'x谢谢QWQ

**Reply by VeryFuns** [reply]
[Uploading QQ录屏20230521135927.zip…]()

**Reply by so1ve** [reply]
文件没有传上来

**Reply by VeryFuns** [reply]
我删了。。。

**Reply by VeryFuns** [reply]
网络没问题 除了forge端都可以进服务器

**Reply by so1ve** [reply]
二分一下试试有没有什么mod冲突了，或者发一下日志

**Reply by VeryFuns** [reply]
OKOK mod没加 日志是这个吗？
[latest.log](https://github.com/Hex-Dragon/PCL2/files/11531478/latest.log)

**Reply by VeryFuns** [reply]
请问还在吗awa

**Reply by so1ve** [reply]
看不出来awa…可能是你服务器不允许forge端进入？你把我的回答取消mark as answer一下好让大佬回答

**Reply by VeryFuns** [reply]
怎么弄 我第一次进github 啥服务器都进不去 比如说hypixel QAQ

**Reply by so1ve** [reply]
怎么把discussion关掉了，，，点一下reopen

**Reply by VeryFuns** [reply]
ok

**Reply by so1ve** [reply]
点一下show previous replies，然后把mark as answer取消掉

**Reply by WorldHim** [reply]
> OKOK mod没加 日志是这个吗？ [latest.log](https://github.com/Hex-Dragon/PCL2/files/11531478/latest.log)

这只是一个崩溃报告。。。

**Reply by VeryFuns** [reply]
az怎么弄

**Comment by VeryFuns** [comment]
可以截图吗 麻烦你了QAQ

**Comment by SilianZ** [comment]
这似乎是服务端与客户端的模组不匹配或者 Forge 和服务端内核的不兼容所导致的。

**Reply by SilianZ** [reply]
鉴于你想要加入的服务器 mcsytt 属于小游戏服务器，服务器可能会因为反作弊等原因不允许 Forge 端加入。
